solr/contrib/ltr/src/java/org/apache/solr/ltr/FeatureLogger.java

```diff
@@ -192,6 +192,33 @@ public String makeFeatureVector(LTRScoringQuery.FeatureInfo[] featuresInfo) {
       return features;
     }
 
+    /*
```

Comment:
Could you explain why you  decided to put these methods public here and not  private in `TestFeatureLogging.java`?